index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Project GitHub Tracker</title>
+		<link rel="stylesheet" href="./style.css" />
+		<script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
+		<link
+			rel="stylesheet"
+			href="https://use.fontawesome.com/releases/v5.12.1/css/all.css"
+			crossorigin="anonymous"
+		/>
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Inconsolata:wght@200;400&display=swap"
+			rel="stylesheet"
+		/>
+	</head>
+	<body>
+		<div class="container">
+			<div class="userimg" id="userimg"></div>
+
+			<div class="space"></div>
+
+			<div class="chart">
+				<canvas id="chart"> </canvas>
+			</div>
+
+			<div class="chartproject" id="chartproject"></div>
+
+			<div class="projectlist" id="projects"></div>
+		</div>
+		<script src="./script.js"></script>
+		<script src="./chart.js"></script>
+	</body>
+</html>
```

Comment:
according to Technigos code guidelines we should use indent with 2 spaces. In your HTML file it looks like you have been using indentation with between 4-6 spaces.